libs/ledger-live-common/src/config/sharedConfig.ts

```diff
@@ -22,11 +22,23 @@ import { tronConfig } from "../families/tron/config";
 import { vechainConfig } from "../families/vechain/config";
 import { appConfig } from "../apps/config";
 
+const countervaluesConfig: ConfigSchema = {
+  countervalues_refreshRate: {
+    type: "number",
+    default: 60 * 1000,
+  },
+  countervalues_marketCapBatchingAfterRank: {
+    type: "number",
+    default: 20,
+  },
+};
+
 const liveCommonConfig: ConfigSchema = {
   ...appConfig,
 };
 
 export const liveConfig: ConfigSchema = {
+  ...countervaluesConfig,
```

Comment:
I think this can be a part of liveCommonConfig above, minor

Reply:
the reasoning behind this is that "countervalues" is a dedicated library and not part of "live-common"

(ultimately the existence of this file that glue things together manually is problematic imo but it's another topic)